src/utils/whatwg-fetch.js

```diff
@@ -0,0 +1,522 @@
+// this file is a copy of https://github.com/github/fetch/tree/v2.0.4
+// the lines marked with PATCH has been changed to make this
```

Comment:
do you need this to work with react-native ? if you're using it in the browser you can use the unpatched package, if you're using it in node I suggest to use node-fetch

Reply:
I agree. I put similar consideration as comments in the function `defaultRetryingFetch` defined in fetch utils. Since some other custom fetches in `fetch utils` relies on it, I left the patched file, but I'm planning to remove it